config.json

```diff
@@ -20,6 +20,13 @@
             "Class" : "DNWS.DelayPlugin",
             "Preprocessing" : "false",
             "Postprocessing" :"false" 
+        },
+        {
+            "Path" : "clientinfo",
+            "Class" : "DNWS.Clientinfo",
```

Comment:
Try to follow the pattern, suffix with Plugin will let you know that this class is a plugin.